src/views/activity/activity.view.tsx

```diff
@@ -82,38 +85,65 @@ const Activity: FC = () => {
   useEffect(() => {
     setWrongNetworkBridges([]);
   }, [connectedProvider?.chainId]);
+  switch (bridgeList.status) {
+    case "pending":
+    case "loading":
+    case "reloading": {
+      return (
+        <>
+          <Header title="Activity" backTo="home" />
+          <PageLoader />
+        </>
+      );
+    }
+    case "failed": {
+      return (
+        <>
+          <Header title="Activity" backTo="home" />
+          <Error error={bridgeList.error} />
```

Comment:
Should we display another error along the snackbar?

Reply:
I thought that an error was better than nothing or a redirect. What would you show?

Reply:
Maybe I would just keep display the spinner and the snackbar. Displaying the same error twice in the screen and in the snackbar looks quite redundant.

Reply:
mmm... a spinner indicates that a process is still in progress, while here, we have an error-ended process. I prefer showing something finale. I agree though that the double error feels redundant (although one could close the snackbar and still have the persistent error):

![image](https://user-images.githubusercontent.com/15896177/173625207-f4c6e717-e216-444c-abc6-15d2bd0c23cd.png)

What about a no-data message in the line of "No bridges available"? This would also be shown when the list is empty and no error occurred (initial state or empty due to the filter).